source/Engine.cpp

```diff
@@ -2192,7 +2266,9 @@ void Engine::DoCollisions(Projectile &projectile)
 		// For weapons with a trigger radius, check if any detectable object will set it off.
 		double triggerRadius = weapon.TriggerRadius();
 		if(triggerRadius)
-			for(const Body *body : shipCollisions.Circle(projectile.Position(), triggerRadius))
+		{
+			vector<Body *> circleCollisions;
+			for(const Body *body : shipCollisions.Circle(projectile.Position(), triggerRadius, circleCollisions))
```

Comment:
this needs to change from iterating over the return of the vector to iterating over the passed result vector

Reply:
The returned vector is what's passed to the function. I made it so I wouldn't have to change so much code.

Unless I messed up and it does an extra copy, that would be bad.